### ISSUE TITLE
Adapter Node

### ISSUE THREAD

**Issue by OlmoDalco** [FractalHQ/sveltekit-electron#17]
Is there a way to use @sveltejs/adapter-node ? 
It seems to works for dev but as soon as i build the app i only get a blank window.

**Comment by FractalHQ** [comment]
Hey- I haven't tried this yet.  Did you manage to get it working?

**Comment by navsubrm** [comment]
Do you have a code example to show the error? I would be willing to take a look at the issue.

**Comment by itssumitrai** [comment]
I have the same issue, the resulting app is empty. Is this project still in development ?

**Comment by ghostdevv** [comment]
Yep! PRs are welcome, wasn't able to get this working in my brief testing before @itssumitrai

**Comment by JBloss1517** [comment]
I'm also running into this problem. Has anyone found a solution to this yet?

**Comment by ghostdevv** [comment]
There are solutions but they aren't worth the trouble IMO - what is your use case? @JBloss1517

**Comment by JBloss1517** [comment]
I was hoping to use the Sveltekit load function (or the new page endpoints) to get my data ahead of the page rendering, and was thinking that was going to require adapter-node to make work.

**Comment by JBloss1517** [comment]
Awesome! Thank you

**Comment by JBloss1517** [comment]
That's so awesome of you, thank you!

**Comment by LukeHagar** [comment]
Hey just wanted to check in on this? I have a use case for the node adapter especially with recent sveltekit updates

**Comment by ghostdevv** [comment]
I started work on an adapter and forgot to finish it! let me see if I can get any further with it this week - and will put what I got online regardless

**Comment by ogrotten** [comment]
Was there a result/solution for this?

**Comment by inday-jin** [comment]
Hey, what use +server.js?
this not working

**Comment by navsubrm** [comment]
I tried working on this, but it wouldn't let me install adapter-node.  Instead, I just created a sample SvelteKit project using adapter-node and published it.  If anyone wants to use it, the link is: https://github.com/navsubrm/svelte-electron-node-adapter/blob/master/index.cjs.

I haven't tested the build on windows, but it builds and runs fine on Mac.  It should work with any imported SvelteKit app.

**Comment by Nate561** [comment]
@ghostdevv, did you ever push the changes you made? I am curious how you solved using the node adapter.

**Comment by JohnRSim** [comment]
Works in win10 ;)

**Comment by ibilux** [comment]
any updates ?

**Comment by JohnRSim** [comment]
> I tried working on this, but it wouldn't let me install adapter-node. Instead, I just created a sample SvelteKit project using adapter-node and published it. If anyone wants to use it, the link is: https://github.com/navsubrm/svelte-electron-node-adapter/blob/master/index.cjs.
> 
> I haven't tested the build on windows, but it builds and runs fine on Mac. It should work with any imported SvelteKit app.

@ibilux I'm using this and building packaging for Mac/Windows and it's working for me - electron/sveltekit with nodejs(sveltekit). 

A few issues I had during working with it.. 
I have to run npm run dev:svelte/electron separately.

On build I have to make sure that sveltekit is not part of the app.asar 
- couldn't seem to get it to work if it was packaged etc..

**Comment by ibilux** [comment]
> I haven't tested the build on windows, but it builds and runs fine on Mac. It should work with any imported SvelteKit app.

I'm trying to use it right now. It's a clever idea. I'll work with it I may can do something with the packaging.

**Comment by JohnRSim** [comment]
If you can figure out the packaging let me know - I wasted a day trying different things to get it to work - and ended up just leaving the build dir unpackaged wiith the config - using nsis for the package manager

```

		"files": [
			"app/**/*",
			"build/**/*",
			"index.cjs",
			"package.json",
			"preload.cjs"
		],
		"asarUnpack": [
			"build/**/*",
			"node_modules/**/*"
		]
```

**Comment by JohnRSim** [comment]
Also forgot to mention I'm using electron-builder for packaging - 
```
npx electron-builder build --windows
npx electron-builder build --mac
```
